addons/pos_loyalty/static/src/overrides/models/pos_store.js

```diff
@@ -563,7 +563,10 @@ patch(PosStore.prototype, {
 
```

Comment:
This method signature is `async processServerData(loadedData)` but the parent does not have parameters, as you do not use the parameter here you can remove it on the signature and make it `async processServerData()`